simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml

```diff
@@ -36,6 +38,7 @@
         <param name="detection_successful_rate" value="1.0"/>
         <param name="enable_ray_tracing" value="false"/>
         <param name="use_object_recognition" value="$(var use_object_recognition)"/>
+        <param name="use_sim_time" value="$(var use_sim_time)"/>
```

Comment:
I personally recommend using `global_parameter_loader`.
https://github.com/autowarefoundation/autoware.universe/issues/143

Or, directly using `set_parameter` is better.

Reply:
@kenji-miyake @takayuki5168 
`global_parameter_loader` package is not added to this package so I think it's better to use use_sim_time argument in this case

Reply:
It's fine, but please do not use `param`. Instead, please use `set_parameter` for global parameters.